dot/network/config.go

```diff
@@ -82,6 +84,8 @@ type Config struct {
 	NoBootstrap bool
 	// NoMDNS disables MDNS discovery
 	NoMDNS bool
+	// ListenAddress multiaddress to listen on
```

Comment:
nit perhaps you meant this:

```suggestion
	// ListenAddress is the multiaddress to listen on
```